modules/data_team_helper/php/endpoints/conflicts.class.inc

```diff
@@ -0,0 +1,287 @@
+                    'visit_label',
+                    'select',
+                    'Visit',
+                    false,
+                    $visit_array
+                ),
+                'SessionID'   => parent::formElementObjectCreator(
+                    'SessionID',
+                    'text',
+                    'SessionID',
+                    true,
+                    null
+                ),
+                'Full_name'   => parent::formElementObjectCreator(
+                    'Full_name',
+                    'text',
+                    'Full_name',
+                    true,
+                    null
+                ),
+                'Instrument'        => parent::formElementObjectCreator(
+                    'Instrument',
+                    'select',
+                    'Instrument',
+                    false,
+                    \Utility::getAllInstruments()
+                ),
+                'Project'           => parent::formElementObjectCreator(
+                    'Project',
+                    'select',
+                    'Project',
+                    true,
+                    \Utility::getProjectList()
+                ),
+                'Site'              => parent::formElementObjectCreator(
+                    'Site',
+                    'select',
+                    'Site',
+                    true,
+                    $list_of_sites
+                ),
+               );
+    }
+
+    /**
+     * Permission verification.
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request.
+     * @param array                  $values  The form values.
+     *
+     * @return array
+     */
+    private function _verifyPermissions(
+        ServerRequestInterface $request,
+        array $values
+    ) : array {
+        $user = $request->getAttribute('user');
+        if (!$user->hasPermission('genomic_browser_view_site')
+            || !$user->hasPermission('genomic_browser_view_allsites')
+        ) {
+            $values['error'] = 'Forbidden';
+        }
+        return $values;
+    }
+
+    /**
+     * An ETagCalculator provides the ability to calculate an ETag for
+     * an incoming HTTP request.
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request.
+     *
+     * @return string The value to use for the ETag header.
+     */
+    public function ETag(ServerRequestInterface $request): string
+    {
+        return md5(json_encode((string) $this->_handleGET($request)->getBody()));
+    }
+
+    /**
+     * Returns true if the user has permission to access
+     * the Genomic Browser module
+     *
+     * @param \User $user The user whose access is being checked
+     *
+     * @return bool true if user has permission
+     */
+    function _hasAccess(\User $user) : bool
+    {
+        return ($user->hasPermission('genomic_browser_view_allsites')
+            || $user->hasPermission('genomic_browser_view_site'));
```

Comment:
I don't think you need `_verifyPermissions` since you have this and it's more standard in LORIS.